.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ repos:
       - id: isort
 
   - repo: https://github.com/psf/black
-    rev: 21.10b0
+    rev: 22.1.0
```

Comment:
> FYI, I've updated this as v22 introduced power operator hugging
> 
> so the list comprehensions kept alternating between `[i**2 for i in range(3)]` and `[i ** 2 for i in range(3)]` depending if I was running black via cli or pre-commit. Might as well update to avoid future confusion smile

Cool - let me open a issue to auto update these ...